README.md

```diff
@@ -49,6 +49,17 @@ resource "lxd_container" "test1" {
 }
 ```
 
+You can also launch a container directly from a remote image, not locally cached, by referencing the remote image name using the format:
```

Comment:
Small nit, but maybe add a note that the remote must already exist and that the `images` is usually enabled by default. It can be verified with `lxc remote list`.